edx_event_bus_kafka/internal/consumer.py

```diff
@@ -143,12 +144,12 @@ def _shut_down(self):
         """
         self._shut_down_loop = True
 
-    def consume_indefinitely(self, offset_timestamp=None):
+    def reset_offsets_indefinitely(self, offset_timestamp):
```

Comment:
I went back and forth on this. It actually will reset the offsets indefinitely -- it's not just once at startup. If the partitions are reassigned at any point while the "consumer" is running, it will reset the offsets again. So in that sense, it really is resetting them "indefinitely". But it's not a great name.

I suppose `reset_offsets_and_sleep_indefinitely` is correct if you parse it as `(reset_offsets_and_sleep)_indefinitely`, but I would tend to read it as `(reset_offsets)_and_(sleep_indefinitely)`, which isn't really correct.

Reply:
So far, I still like `reset_offsets_and_sleep_indefinitely` the best. You could add a note to the docstring explaining that the name is imperfect, because it will also reset again if partitions are reassigned, etc. But, this name feels closer to one that would keep me from looking deeper unless I needed to.

Reply:
I'm OK making the change.